src/components/Modal/BaseModal.tsx

```diff
@@ -50,6 +50,7 @@ function BaseModal(
         shouldEnableNewFocusManagement = false,
         restoreFocusType,
         shouldUseModalPaddingStyle = true,
+        initialFocus,
```

Comment:
Can we initialize `initialFocus` to `undefined` by default? I think that would be neater.